### PR TITLE
Aladdin generated examples for Network

### PR DESCRIPTION
**Pull request by aladdindoc**
- Add Aladdin crafted examples.

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Network/Network/help/Add-AzLoadBalancerProbeConfig.md [hunk 1]
```diff
@@ -24,13 +24,22 @@ The **Add-AzLoadBalancerProbeConfig** cmdlet adds a probe configuration to an Az
...
 
 This command gets the load balancer named myLb, adds the specified probe configuration to it, and then uses the **Set-AzLoadBalancer** cmdlet to update the load balancer.
 
+### Example 2
+
+Adds a probe configuration to a load balancer. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+Add-AzLoadBalancerProbeConfig -IntervalInSeconds 15 -LoadBalancer <PSLoadBalancer> -Name 'NewProbe' -Port 80 -ProbeCount 2 -Protocol 'https'
```
Comment (mikefrobbins) [inline src/Network/Network/help/Add-AzLoadBalancerProbeConfig.md:40]: I find it odd that port 80 would be set to HTTPS. By default, port 80 is HTTP and port 443 is HTTPS. A different port might be used for testing, but I've never seen anyone run SSL across port 80.

File: src/Network/Network/help/New-AzApplicationGatewayBackendHttpSetting.md [hunk 2]
```diff
@@ -30,13 +30,22 @@ Back-end HTTP settings are applied to all back-end servers in a pool.
...
 This command creates back-end HTTP settings named Setting01 on port 80, using the HTTP protocol, with cookie-based affinity disabled.
 The settings are stored in the $Setting variable.
 
+### Example 2
+
+Creates back-end HTTP setting for an application gateway. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+New-AzApplicationGatewayBackendHttpSetting -CookieBasedAffinity Enabled -Name 'Setting01' -PickHostNameFromBackendAddress -Port 80 -Probe <PSApplicationGatewayProbe> -Protocol https -RequestTimeout <Int32>
```
Comment (mikefrobbins) [inline src/Network/Network/help/New-AzApplicationGatewayBackendHttpSetting.md:46]: Same here. Seems odd to use port 80 with HTTPS (SSL).

File: src/Network/Network/help/New-AzFirewallPolicyApplicationRule.md [hunk 3]
```diff
@@ -38,6 +38,15 @@ PS C:\> New-AzFirewallPolicyApplicationRule -Name AR1 -SourceAddress "192.168.0.
 
 This example creates an application rule with the source address, protocol and the target fqdns.
 
+### Example 2
+
+Create a new Azure Firewall Policy Application Rule. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+New-AzFirewallPolicyApplicationRule -Description 'Allow RDP' -Name AR1 -Protocol 'http:80' -SourceAddress '192.168.0.0/16' -TargetFqdn '*.ro'
```
Comment (mikefrobbins) [inline src/Network/Network/help/New-AzFirewallPolicyApplicationRule.md:47]: The value for description or protocol seems inaccurate. RDP defaults to port 3389. Protocol specifies http:80 with is used by non-SSL web servers.

File: src/Network/Network/help/New-AzLoadBalancerProbeConfig.md [hunk 4]
```diff
@@ -25,13 +25,22 @@ The **New-AzLoadBalancerProbeConfig** cmdlet creates a probe configuration for a
...
 This command creates a probe configuration named MyProbe using the HTTP protocol.
 The new probe will connect to a load-balanced service on port 80.
 
+### Example 2
+
+Creates a probe configuration for a load balancer. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+New-AzLoadBalancerProbeConfig -IntervalInSeconds 15 -Name 'MyProbe' -Port 80 -ProbeCount 15 -Protocol 'https' -RequestPath 'healthcheck.aspx'
```
Comment (mikefrobbins) [inline src/Network/Network/help/New-AzLoadBalancerProbeConfig.md:41]: Same. Port 80 isn't typically associated with HTTPS.

File: src/Network/Network/help/New-AzVpnSiteLink.md [hunk 5]
```diff
@@ -45,6 +45,15 @@ PS C:\> $vpnSite = New-AzVpnSite -ResourceGroupName "testRG" -Name "testVpnSite"
 
 The above will create a resource group, Virtual WAN and a VpnSite with 1 VpnSiteLinks in West US in "testRG" resource group in Azure.
 
+### Example 2
+
+Creates an Azure VpnSiteLink object. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+New-AzVpnSiteLink -BGPAsn <UInt32> -BGPPeeringAddress <String> -IPAddress '15.25.35.45' -LinkProviderName 'SomeTelecomProvider' -LinkSpeedInMbps '10' -Name 'testVpnSiteLink1'
```
Comment (mikefrobbins) [inline src/Network/Network/help/New-AzVpnSiteLink.md:54]: Not sure that I like a real public IP address being used that isn't owned by Microsoft. Is this real customer data?

File: src/Network/Network/help/Set-AzRouteConfig.md [hunk 6]
```diff
@@ -64,6 +64,15 @@ This command gets the route table named RouteTable01 by using the Get-AzRouteTab
...
 The current cmdlet modifies the route named Route02, and then passes the result to the **Set-AzRouteTable** cmdlet, which updates the table to reflect your changes.
 
+### Example 2
+
+Updates a route configuration for a route table. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+Set-AzRouteConfig -AddressPrefix <String> -Name 'cert01' -NextHopIpAddress '12.0.0.5' -NextHopType 'VnetLocal'
+```
```
Comment (dcaro) [inline src/Network/Network/help/Set-AzRouteConfig.md:74]: This example is missing the required parameter `-RouteTable` that can also be passed through piping.

**Comment by aladdindoc** [comment]
#sign-off

**Pull request merged**